src/main/java/io/fusionauth/domain/api/APIKeyResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019, FusionAuth, All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package io.fusionauth.domain.api;
+
+import com.inversoft.json.JacksonConstructor;
+import io.fusionauth.domain.AuthenticationAPIKey;
+
+/**
+ * Authentication key response object.
+ *
+ * @author Sanjay
+ */
+public class APIKeyResponse {
+  public String authenticationKeyId;
+
+  public AuthenticationAPIKey apiKey;
+
+  @JacksonConstructor
+  public APIKeyResponse() {
+  }
+
+  public APIKeyResponse(String authenticationKeyId) {
+    this.authenticationKeyId = authenticationKeyId;
+  }
+
+  public APIKeyResponse(AuthenticationAPIKey authenticationKey) {
+    this.apiKey = authenticationKey;
+  }
```

Comment:
By convention, we usually make the parameter equal in name to the property. 

And for consistency these two fields should probably be named something similar, we could do :


```
public String authenticationKeyId;

public AuthenticationAPIKey authenticationKey;
```

or 

```
public String keyId;

public AuthenticationAPIKey key;
```

I would look at the JSON and see which ones feels cleaner, more concise, and descriptive enough.